aep/general/0193/aep.md.j2

```diff
@@ -1,5 +1,150 @@
 # Errors
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/48) for more
-information.
+Effective error communication is an important part of designing simple and
+intuitive APIs. Services returning standardized error responses enable API
+clients to construct centralized common error handling logic. This common logic
+simplifies API client applications and eliminates the need for cumbersome
+custom error handling code.
+
+## Guidance
+
+Services **must** clearly distinguish successful responses from error
+responses.
+
+### Structure
+
+An error response **should** contain the following fields:
```

Comment:
This guidance is incompatible with the (correct and unavoidable) recommendation to use `google.rpc.Status` for protobuf below (assuming ~all protobuf APIs are using gRPC, anyway).

I don't think it's necessarily desirable for gRPC to dictate how non-gRPC errors should work, but it's also not desirable for us to have inconsistent guidance, nor to have guidance that gRPC APIs can't implement.

Reply:
As discussed in the meeting on 3/29, we'll let gRPC APIs pass the problem details structure in the `details` field of the Status message.